labels-annotation-for-openshift.adoc

```diff
@@ -113,6 +113,11 @@ TIP: Labels must be applied to ALL resources, DeploymentConfigs, BuildConfigs, S
 | Identifier for the version of the source code. Can be branch, tag or commit SHA for the application. If omitted master head of the default branch is assumed
 | v1.0.0
 
+| app.openshift.io/connects-to:
+| OPT
+| The component that this component is connecting to (e.g. a component making API calls to another `service` ) 
+| e.g. Component `front-end` has a `connects-to` which has a value `backend`
+
```

Comment:
I am not sure if we are clear about what value is expected on this definition. Is it `app.kubernetes.io/instance` or `app.kubernetes.io/name` or a combination of both?